.github/workflows/release.yml

```diff
@@ -404,10 +404,41 @@ jobs:
             const { owner, repo } = context.repo
             await github.rest.repos.updateRelease({ owner, repo, release_id: ${{ needs.set-release-id.outputs.release_id }}, prerelease: false })
 
+  pyk-publish:
+    name: 'Publish pyk'
+    runs-on: ubuntu-latest
+    environment: production
+    permissions:
+      id-token: write
```

Comment:
I wonder, is this part necessary? @Baltoli @F-WRunTime

Reply:
The secret is porbably defined for the `production` environment.

The other part is apparently for signing the published package: https://github.com/pypa/gh-action-pypi-publish/blob/unstable/v1/README.md?plain=1#L63-L64

Reply:
Sorry seeing this now, but yes it is necessary. I want to transition to using a verified publisher approach this sets us up for that. And yes the secret is stored in that environment. It's ultimately a better practice than storing everything in a general environment.